tools/Makefile

```diff
@@ -0,0 +1,6 @@
+# valid for Debian Buster
+install-dummy_hcd:
+	sudo ./install-dummy_hcd.sh
+
+create-hid-gadget:
```

Comment:
we can also have a "destroy" target which removes the hid gadget; it may be calling the same shell script but with different arguments